src/Compilers/Server/VBCSCompiler/BuildServerController.cs

```diff
@@ -140,21 +140,25 @@ internal static ICompilerServerHost CreateCompilerServerHost(ICompilerServerLogg
             // exit immediately with a non-zero exit code
             var mutexName = BuildServerConnection.GetServerMutexName(pipeName);
             bool createdNew;
-            using (var serverMutex = BuildServerConnection.OpenOrCreateMutex(name: mutexName,
-                                                                             createdNew: out createdNew))
+            var serverMutex = BuildServerConnection.OpenOrCreateMutex(name: mutexName,
+                                                                             createdNew: out createdNew);
+            if (!createdNew)
+            {
+                return CommonCompiler.Failed;
```

Comment:
This is not a fatal error. It just means another thread has opened the mutex in parallel.